src/handlers/request/tests/RequestHandlers.spec.ts

```diff
@@ -0,0 +1,62 @@
+import {RequestHandler} from '..';
+import handleRequest from '../RequestHandlers';
+
+describe('RequestHandlers', () => {
```

Comment:
How does your code work if the `canProcess` or `process` attributes are not present?